test/python/test_ibmqjob.py

```diff
@@ -102,11 +104,9 @@ def test_run_device(self):
         qobj = transpiler.compile(self._qc, backend)
         shots = qobj.config.shots
         job = backend.run(qobj)
-        while not (job.done or job.exception):
-            self.log.info(job.status)
+        while not (job.status() == JobStatus.DONE):
```

Comment:
Non-blocking: what about using `is` instead of `==` as in `job.status() is JobStatus.DONE`, it is the [way preferred in the docs](https://docs.python.org/3/library/enum.html#comparisons), it is also faster and more meaningful. If you go for it, update all the references.